.github/workflows/wipac-cicd.yml

```diff
@@ -101,6 +112,7 @@ jobs:
 
   docker:
     name: "Docker Image"
+    if: ${{ github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main' }}
     needs: [flake8, pip-install, py-tests]
```

Comment:
you'll want to "need" `py-setup` in the off case you have a very slow `py-setup`

Reply:
Ah, right.  I thought there was still a connection there.